homeassistant/components/plaato/config_flow.py

```diff
@@ -1,10 +1,212 @@
-"""Config flow for GPSLogger."""
-from homeassistant.helpers import config_entry_flow
+"""Config flow for Plaato."""
+import logging
 
-from .const import DOMAIN
+from pyplaato.plaato import PlaatoDeviceType
+import voluptuous as vol
 
-config_entry_flow.register_webhook_flow(
-    DOMAIN,
-    "Webhook",
-    {"docs_url": "https://www.home-assistant.io/integrations/plaato/"},
+from homeassistant import config_entries
+from homeassistant.const import CONF_TOKEN, CONF_WEBHOOK_ID
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+
+from ...config_entries import ConfigEntry
+from .const import (
+    CONF_CLOUDHOOK,
+    CONF_DEVICE_NAME,
+    CONF_DEVICE_TYPE,
+    CONF_UPDATE_INTERVAL,
+    CONF_USE_WEBHOOK,
+    DOCS_URL,
+    PLACEHOLDER_DEVICE_NAME,
+    PLACEHOLDER_DEVICE_TYPE,
+    PLACEHOLDER_DOCS_URL,
+    PLACEHOLDER_WEBHOOK_URL,
 )
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__package__)
+
+
+class PlaatoConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handles a Plaato config flow."""
+
+    VERSION = 2
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Initialize."""
+        self._init_info = {}
+        self._errors = {}
+
+    async def async_step_user(self, user_input=None):
+        """Handle user step."""
+        return self.async_show_form(
+            step_id="device_type",
```

Comment:
```suggestion
```

The step id should match to the step name.  Move to the next step once the form is completed.  Please see the examples in `harmony/config_flow.py`